<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewNotebook</class>
 <widget class="QDialog" name="NewNotebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>173</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>-1</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_picture">
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_new_notebook">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>New Notebook</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_name">
       <property name="text">
        <string>Enter a name for the new notebook :  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_name">
       <property name="placeholderText">
        <string>Notebook name</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Project : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbx_project"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_create">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
